### PR TITLE
Downgrade husky due to change of license #13940

### PR DESCRIPTION
**Pull request by PierreBesson**
---

Please make sure the below checklist is followed for Pull Requests.

- [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green

When you are still working on the PR, consider converting it to Draft (bellow reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

**Review by murdos — APPROVED** [review]

**Comment by mshima** [comment]
Please disable it at dependabot https://github.com/jhipster/generator-jhipster/blob/401ec1755cc1d6e65bc76cb99c07211d1de83c1d/.github/dependabot.yml#L23

File: .github/dependabot.yml [hunk 1]
```diff
@@ -27,6 +27,10 @@ updates:
     versioning-strategy: increase
     labels:
       - 'theme: dependencies'
+    ignore:
+      # Ignore updates to husky due to the license change
+      # See https://github.com/jhipster/generator-jhipster/issues/13940
+      - dependency-name: 'husky'
```
Comment (mshima) [inline .github/dependabot.yml:33]:
```suggestion
      - dependency-name: 'husky'
        versions: ["5.x"]
```

**Review by mshima — COMMENTED** [review]
v4 is open source, so let's ignore v5 only for now.

**Comment by pascalgrimaud** [comment]
There are some prettier error with this PR

**Comment by mshima** [comment]
@pascalgrimaud just saw it. Fixed.

**Comment by pascalgrimaud** [comment]
You're so fast @mshima ! Thanks

**Pull request merged**